src/charts/bar.js

```diff
@@ -793,12 +795,35 @@ define(function(require) {
             return this;
         };
 
+        /**
+         * Gets or Sets the highlightBarFunction function. The callback passed to
+         * this function returns a bar component from the bar chart. Use this function
+         * if you want to apply a custom behavior to the highlighted bar on hover.
+         * When hasSingleBarHighlight is true the highlighted bar will be the
+         * one that was hovered by the user. When hasSingleBarHighlight is false
+         * the highlighted bars are all the bars but the hovered one. The default
+         * highlight effect on a bar is darkening the highlighted bar(s) color.
+         * @param  {Function} _x        Desired operation operation on a hovered bar passed through callback
+         * @return {highlightBarFunction | module} Is highlightBarFunction used or Chart module to chain calls
+         * @public
+         * @example barChart.highlightBarFunction(e => d3.select(e).attr('fill', 'blue'))
+         * barChart.highlightBarFunction(e => null) // will disable the default highlight effect
+         */
+        exports.highlightBarFunction = function(_x) {
+            if (!highlightBarFunction.length) {
+                return highlightBarFunction;
+            }
+            highlightBarFunction = _x;
+
+            return this;
+        }
+
         /**
          * Gets or Sets the isAnimated property of the chart, making it to animate when render.
          * By default this is 'false'
          *
          * @param  {Boolean} _x Desired animation flag
-         * @return { isAnimated | module} Current isAnimated flag or Chart module
```

Comment:
Thought it would be a good idea to remove those spaces, seen on several occasions

Reply:
Go for it. 
We haven't been super strict with all that formating.